packages/core/useElementBounding/index.ts

```diff
@@ -91,6 +92,10 @@ export function useElementBounding(
 
   useResizeObserver(target, update)
```

Comment:
useMutationObserver can replace useResizeObserver.

Reply:
Apologies, the MutationObserver cannot observe additions to the subtree, which impacts the size of the DOM. Therefore, we cannot remove the use of ResizeObserver. You may need to roll back to the previous code.